### PR TITLE
Made minor spacing and tab edits.

### PR DESCRIPTION
**Pull request by juliemtaylor**
Branch is to start a code review for this component.

### CODE REVIEW

File: new-components/headline-component.html [hunk 1]
```diff
@@ -185,19 +192,23 @@
                                 <div class="headline-component-heading">Customize the Ideal Vacation</div>
                                 <div class="headline-component-description">Delta Vacations gives the benefits and flexibility to truely own the vacation experience with packages inspired by hundreds of destinations and thousands of accomodations and endless activites</div>
                                 <div class="headline-component-button"><button class="headline-component-learn-more"> Learn More</button></div>
+
                         </div>
                         <div class="headline-component-heading-description-arrow headline-component-hide">
                                 <div class="headline-component-heading">Customize the Ideal Vacation</div>
                                 <div class="headline-component-description">Delta Vacations gives the benefits and flexibility to truely own the vacation experience with packages inspired by hundreds of destinations and thousands of accomodations and endless activites</div>
                                 <div class="headline-component-arrow"><img src=" CACR/images/arrow.png" /></div>
+
```
Comment (juliemtaylor) [inline new-components/headline-component.html]: Same question here, not sure the `<div>` is needed since the class can be applied to the ``.

File: new-components/headline-component.html [hunk 2]
```diff
@@ -185,19 +192,23 @@
                                 <div class="headline-component-heading">Customize the Ideal Vacation</div>
                                 <div class="headline-component-description">Delta Vacations gives the benefits and flexibility to truely own the vacation experience with packages inspired by hundreds of destinations and thousands of accomodations and endless activites</div>
                                 <div class="headline-component-button"><button class="headline-component-learn-more"> Learn More</button></div>
+
```
Comment (juliemtaylor) [inline new-components/headline-component.html]: Why wrap the `<button>` in a `<div>` here?

File: new-components/headline-component.html [hunk 3]
```diff
@@ -173,6 +178,8 @@
 });
 </script>
 
+
```
Comment (juliemtaylor) [inline new-components/headline-component.html]: Wondering why you adding the hyperlinks with javascript. Is this to simplify the code on the AEM side?

File: new-components/headline-component.html [hunk 4]
```diff
@@ -30,6 +33,7 @@
     font-size: 30px;
     font-weight: 600;
     font-family: Open Sans;
+
```
Comment (juliemtaylor) [inline new-components/headline-component.html]: Same note as above adding quotes to font name that contain spaces.

File: new-components/headline-component.html [hunk 5]
```diff
@@ -20,6 +22,7 @@
     bottom: 0px;
     background-color: rgba(0, 0, 0, 0.8);
     font-family: open sans;
```
Comment (juliemtaylor) [inline new-components/headline-component.html]: Using quotes around font names that contain spaces will prevent errors. Also, using the same capitalization as in the font style sheet is a good idea.

font-family: 'Open Sans';

Comment (PradeepaJayaprakash) [inline new-components/headline-component.html]: This has been fixed. Wrapped the font names in quotes and also took care of capitalization

File: new-components/headline-component.html (hunk 4 above)
Comment (PradeepaJayaprakash) [inline new-components/headline-component.html]: This is done.

File: new-components/headline-component.html (hunk 3 above)
Comment (PradeepaJayaprakash) [inline new-components/headline-component.html]: This was a review comment from chris to write a javascript for onclick event and listen to the click of the outer container class.

File: new-components/headline-component.html [hunk 6]
```diff
@@ -3,6 +3,8 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
 </head>
 
+
+
```
Comment (juliemtaylor) [inline new-components/headline-component.html]: As we discussed on HipChat :) 
On line 13 add
.v2 .headline-component:hover .headline-component-image {
    opacity: 0.8;
}
.v2 .headline-component:hover button {
  background-color: #003366;
    cursor: pointer;
}

Added the cursor because I noticed the button was an arrow when hovered.

File: new-components/headline-component.html (hunk 3 above)
Comment (juliemtaylor) [inline new-components/headline-component.html]: Ah, ok. Well wrapping the whole banner in an `<a>` would be what I would do here. It's not a problem to do that now with HTML5 specifications.

File: new-components/headline-component.html (hunk 2 above)
Comment (PradeepaJayaprakash) [inline new-components/headline-component.html]: Implemented the review comments

File: new-components/headline-component.html [hunk 7]
```diff
@@ -39,7 +46,7 @@
 .v2 .headline-component-description {
     font-size: 14px;
     font-weight: 300;
-    font-family: Open sans;
+    font-family: 'Open Sans';
     color: #ffffff;
     text-align: left;
     padding: 15px 0px;
```
Comment (orderedlist) [inline new-components/headline-component.html]: Unless you're specifically resetting or overriding the top and bottom padding with this attribute, I'd prefer to specify the left and right on their own. Eg. `padding-left: 15px; padding-right: 15px;`.
